vllm/worker/selector.py

```diff
@@ -0,0 +1,26 @@
+from vllm.config import DeviceConfig
```

Comment:
The code looks fine, please rebase and test if multistep scheduling is not broken with this PR

Reply:
Codes rebased to latest habana_main